examples/experimental/dagster-airlift/dagster_airlift/in_airflow/base_proxy_operator.py

```diff
@@ -135,31 +169,14 @@ def launch_runs_for_task(self, context: Context, dag_id: str, task_id: str) -> N
             logger.debug(
                 f"Triggering run for {repo_location}/{repo_name}/{job_name} with assets {asset_keys}"
             )
-            response = session.post(
-                f"{dagster_url}/graphql",
-                json={
-                    "query": TRIGGER_ASSETS_MUTATION,
-                    "variables": {"executionParams": execution_params},
-                },
-                # Timeout in seconds
-                timeout=10,
-            )
-            launch_data = self.get_valid_graphql_response(response, "launchPipelineExecution")
-            run_id = launch_data["run"]["id"]
-            logger.debug(f"Launched run {run_id}...")
+            run_id = self.launch_dagster_run(context, session, dagster_url, execution_params)
             triggered_runs.append(run_id)
         completed_runs = {}  # key is run_id, value is status
         while len(completed_runs) < len(triggered_runs):
             for run_id in triggered_runs:
                 if run_id in completed_runs:
                     continue
-                response = session.post(
-                    f"{dagster_url}/graphql",
-                    json={"query": RUNS_QUERY, "variables": {"runId": run_id}},
-                    # Timeout in seconds
-                    timeout=3,
-                )
-                run_status = self.get_valid_graphql_response(response, "runOrError")["status"]
+                run_status = self.get_dagster_run_status(session, dagster_url, run_id)
```

Comment:
Suggestion: A naming convention that makes it clear this will result in an API/network call.
`get` can imply something "cheap".

`query_dagster_run_status` for example gives this appropriate weight?